GTG/gtk/colors.py

```diff
@@ -155,15 +155,12 @@ def get_colored_tags_markup(ds, tag_names):
 
 def generate_tag_color():
 
-    maxvalue = 1.0
```

Comment:
The original error was that `maxvalue` is unused. The loop was written basically in C, so I wrote something more Pythonic. I checked that tag color generation still works.